wondrous-app/components/Common/ProfilePictureHelpers/index.tsx

```diff
@@ -1,20 +1,25 @@
+import { useMemo } from 'react';
+
 import { DAOIcon } from 'components/Icons/dao';
 import { NoLogoDAO } from 'components/Common/SidebarMain/styles';
-import { SafeImage } from '../Image';
-import DefaultUserImage from '../Image/DefaultUserImage';
+import { SafeImage, DefaultUserImage } from 'components/Common/Image';
 
 export const OrgProfilePicture = ({ profilePicture, style = {} }) => {
+  const imageStyles = useMemo(
```

Comment:
We definitely need to replace inline-styles for something memoized, but I don't like this way like putting styles here inside the component. Maybe it's better to move object with these styles to `ProfilePictureHelpersStyles` file and in component just merge it with props styles? For EntityMenu we can do the same - move styles object to styles file and pass it to props. What do you think?